src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Drawable {
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    } // I need this getter to get color in overrided method "draw" in every figure class
```

Comment:
We know, remove the comment :) 
If you want to clarify smth better do it in the commit messages than in the actual source code